__test__/manage-project.test.js

```diff
@@ -0,0 +1,123 @@
+const { By, until, Key } = require('selenium-webdriver-3');
+const { getOtp } = require('./getotp');
```

Comment:
I think Linux systems are case sensitive regarding file names, so this should be `getOtp` for being able to work with our GitHub actions.